libs/@local/hash-isomorphic-utils/.eslintrc.cjs

```diff
@@ -29,5 +29,12 @@ module.exports = {
         ],
       },
     },
+    {
+      files: ["./src/system-types/*.ts"],
+      rules: {
+        "simple-import-sort/exports": "off",
+        "simple-import-sort/imports": "off",
+      },
```

Comment:
Is this to avoid eslint errors after running `yarn generate-system-types`? If so maybe we can just run `yarn fix:eslint` as part of that script, as these should be auto-fixable errors?

Reply:
Done in https://github.com/hashintel/hash/pull/2852/commits/eb32f8c6e2e7ed6f76d88520d1b1fd58e5d0edf3

Reply:
The ordering of properties is still non-deterministic. I'll worry about this in follow up.